test/k8s-integration/driver-config.go

```diff
@@ -124,3 +138,38 @@ func generateDriverConfigFile(platform, pkgDir, storageClassFile, snapshotClassF
 
 	return configFilePath, nil
 }
+
+func getGKEClusterVersion() (major, minor int, err error) {
+	if *gkeClusterVer == "" && *gkeReleaseChannel == "" {
+		return 0, 0, fmt.Errorf("No GKE version found")
+	}
+	// TODO: the release channels are hardcoded (as is latest, below), which means they will need to be updated at each GKE release. It would be better to find the version automatically, maybe through gcloud container get-server-config.
```

Comment:
I thought @saikat-royc had implemented something that automatically gets the version of the release channel?

Reply:
https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/master/test/k8s-integration/version.go

Reply:
Also instead of statically mapping release channel to a cluster version(as it can change), we can do something similar to the logic of generating skip string, where we read the k8s cluster version after the cluster is deployed and use mustParseVersion() to parse the gke version (since we know the deployment strategy), and compare versions. generateDriverConfigFile comes after cluster is deployed
[here](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/12ea362bb378bb52fbbbb064407889719092ffea/test/k8s-integration/main.go#L421) and [here](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/12ea362bb378bb52fbbbb064407889719092ffea/test/k8s-integration/main.go#L427)

Reply:
Ah, thank you, I wrote this PR quickly and didn't have time to figure out exactly when this was called, it looked like it was  early.

If the cluster is up than your suggestion is good, I'll do that. Thanks for saving me the trouble of figuring out the sequencing :-)

Reply:
Indeed you're right, done.